tests/gen_many_js_functions.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+NUM_FUNCS_TO_GENERATE = 1000
+
+def func_name(i):
+	return 'thisIsAFunctionWithVeryLongFunctionNameThatWouldBeGreatToBeMinifiedWhenImportingToAsmJsOrWasmSideCodeToCallOtherwiseCodeSizesWillBeLargeAndNetworkTransfersBecomeVerySlowThatUsersWillGoAwayAndVisitSomeOtherSiteInsteadAndThenWebAssemblyDeveloperIsSadOrEvenWorseNobodyNoticesButInternetPipesWillGetMoreCongestedWhichContributesToGlobalWarmingAndThenEveryoneElseWillBeSadAsWellEspeciallyThePolarBearsAndPenguinsJustThinkAboutThePenguins' + str(i+1)
+
+def generate_js_library_with_lots_of_functions(out_file):
+	f = open(out_file, 'w')
```

Comment:
Maybe use `with open(out_file, 'w') as f:` and then drop the `close()`, here and below